great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -678,8 +678,8 @@ def get_domain_records(  # noqa: C901 - 24
                     .where(
                         sa.not_(
                             sa.and_(
-                                sa.column(column_A_name) is None,
-                                sa.column(column_B_name) is None,
+                                sa.column(column_A_name) == None,  # noqa: E711
+                                sa.column(column_B_name) == None,  # noqa: E711
```

Comment:
Ended up having to revert the `x is None` change.
The `column` `==`/`__eq__` operator must have some logic that returns `None`.